docs/includes/generated_docs/schemas/tpp.md

```diff
@@ -2357,6 +2357,69 @@ The Healthcare Resource Group (HRG) code assigned to the activity, used to assig
   </dd>
 </div>
 
+<div markdown="block">
+  <dt id="opa.outcome_of_attendance">
+    <strong>outcome_of_attendance</strong>
+    <a class="headerlink" href="#opa.outcome_of_attendance" title="Permanent link">🔗</a>
+    <code>string</code>
+  </dt>
+  <dd markdown="block">
+This records the outcome of an Out-Patient Attendance Consultant.Refer to the [NHS Data Model and Dictionary entry for "outcome of attendance"](https://www.datadictionary.nhs.uk/attributes/outcome_of_attendance.html) andthe [HES Data Dictonary](https://digital.nhs.uk/data-and-information/data-tools-and-services/data-services/hospital-episode-statistics/hospital-episode-statistics-data-dictionary#nhs-data-model-and-dictionary)for details on code meanings.
```

Comment:
"the outcome of an Out-Patient Attendance Consultant" reads like a typo?

Reply:
It is; not sure what's happened there.

Reply:
Oh, no it's not - it comes from the NHS data dictionary, but I obviously didn't actually read it in my head
https://www.datadictionary.nhs.uk/attributes/outcome_of_attendance.html

The HES data dictonary description is "The outcome of an outpatient attendance", which sounds better - I'll change it to that